modules/indexer/issues/internal/model.go

```diff
@@ -74,6 +74,8 @@ type SearchResult struct {
 type SearchOptions struct {
 	Keyword string // keyword to search
 
+	IsFuzzyKeyword bool // if false the levenshtein distance is 0
```

Comment:
Am I missing something, or is this new field never set?

Reply:
> ... For future works I added an option in backend only atm, to enable fuzzynes for issue indexer too ...

yes as we currently want & use the default case `false`

later the UI should have an option to set it to true, but that's no bugfix anymore